frontend/src/constants/localStorage.ts

```diff
@@ -12,3 +12,8 @@ export const LOGGED_IN_KEY = 'is-logged-in'
  * been modified.
  */
 export const LOCAL_STORAGE_EVENT = 'local-storage'
+
+/**
+ * Key to store whether a user has seen the rollout announcements before.
+ */
+export const ROLLOUT_ANNOUNCEMENT_KEY = 'has-seen-rollout-announcement'
```

Comment:
hmmm...what if two users use the same browser? only the first one will see the announcement right?

Reply:
Yes only the first one will see. I've updated the key to be suffixed with user's ID, this change solves this problem